src/CMakeLists.txt

```diff
@@ -22,10 +22,16 @@ set_target_properties(benchmark PROPERTIES
   VERSION ${GENERIC_LIB_VERSION}
   SOVERSION ${GENERIC_LIB_SOVERSION}
 )
+
+include(GenerateExportHeader)
+generate_export_header(benchmark EXPORT_FILE_NAME ${CMAKE_CURRENT_SOURCE_DIR}/../include/benchmark_export.h)
```

Comment:
It's a very bad idea to write outside of the `CMAKE_CURRENT_BINARY_DIR`

Reply:
thanks, I'll try to change to use CMAKE_CURRENT_BINARY_DIR. initial attempt produced some CMake errors for me. also, build fails hard on tests, I'll take a look.